great_ape_safe/ape_api/ebtc.py

```diff
@@ -1606,3 +1608,86 @@ def cdp_withdraw_debt(self, cdp_id, debt_withdrawable_amount):
 
         # 2.2. icr decreased
         assert new_icr < prev_icr
+
+    def adjust_cdp_with_collateral(
```

Comment:
I think we should add a test for this function - I can push in virtue of time and as part of my review.

Reply:
tests were added in this pr https://github.com/ebtc-protocol/ebtc-multisig/pull/46